packages/clerk-js/src/core/resources/OrganizationMembership.ts

```diff
@@ -71,6 +72,22 @@ export class OrganizationMembership extends BaseResource implements Organization
     this.updatedAt = unixEpochToDate(data.updated_at);
     return this;
   }
+
+  public async reload(params?: ClerkResourceReloadParams): Promise<this> {
+    const { rotatingTokenNonce } = params || {};
+    const json = await BaseResource._fetch(
+      {
+        method: 'GET',
+        path: `/me/organization_memberships`,
+        rotatingTokenNonce,
+      },
+      { forceUpdateClient: true },
+    );
+    const currentMembership = (json?.response as unknown as OrganizationMembershipJSON[]).find(
+      orgMem => orgMem.id === this.id,
+    );
+    return this.fromJSON(currentMembership as OrganizationMembershipJSON);
```

Comment:
Can we make sure the `useOrganization` hook returns a new value when this is reloaded?

Reply:
@nikosdouvlis @desiprisg As far as I've tested until now, it does work as expected.